ironfish/src/migrations/data/031-add-pak-to-account/new/AccountValue.ts

```diff
@@ -53,8 +51,9 @@ export class AccountValueEncoding implements IDatabaseEncoding<AccountValue> {
     }
 
     if (value.multisigKeys) {
-      bw.writeVarBytes(Buffer.from(value.multisigKeys.secret, 'hex'))
-      bw.writeVarBytes(Buffer.from(value.multisigKeys.keyPackage, 'hex'))
+      const encoding = new MultisigKeysEncoding()
+      bw.writeU64(encoding.getSize(value.multisigKeys))
+      bw.writeBytes(encoding.serialize(value.multisigKeys))
     }
```

Comment:
Does this break the current multisig accounts because of the size also being written?

Reply:
Maybe this change doesn't matter because there were no multisig accounts before 31?

Reply:
Yes that's correct. If this ran for anyone in the first place, their accounts would not be de-codable. It would crash when they try to start the node.